components/gateway/internal/audit/messages/audit.go

```diff
@@ -50,9 +54,33 @@ func NewAuditMessagePayload(
 	request HttpRequest,
 	response HttpResponse,
 ) publish.EventMessage {
+	identity := ""
+
+	if request.Header != nil {
+		tokenString := strings.Replace(request.Header.Get("Authorization"), "Bearer ", "", 1)
+		token, _, err := new(jwt.Parser).ParseUnverified(tokenString, jwt.MapClaims{})
+		if err != nil {
+			_ = fmt.Errorf("error for Parse %s", err)
+		}
+		if token != nil {
+			if claims, ok := token.Claims.(jwt.MapClaims); ok {
+				identity = fmt.Sprint(claims["sub"])
+			} else {
+				fmt.Printf("error get claims JWT token: %s", err)
+				fmt.Printf("\n")
+			}
+		}
+
+		request.Header.Del("Authorization")
+	}
+
+	if request.Path == "/api/auth/oauth/token" {
```

Comment:
Not a big deal, but you should get this url from the openid connect discovery endpoint